package.json

```diff
@@ -1,52 +1,61 @@
 {
   "name": "app_scaffold",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "A scaffold for developers to build ZAF v2 apps",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/zendesk/app_scaffold.git"
-  },
-  "scripts": {
-    "build": "webpack -p",
-    "watch": "webpack --watch"
-  },
-  "license": "Apache-2.0",
-  "author": "Zendesk",
   "keywords": [
     "zendesk",
     "app",
     "framework"
   ],
-  "bugs": {
-    "url": "https://github.com/zendesk/app_scaffold/issues"
+  "author": "Zendesk",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.3.0"
+  },
+  "scripts": {
+    "test": "jest",
+    "watch": "cross-env NODE_ENV='development' webpack --watch --mode=development",
+    "build:dev": "cross-env NODE_ENV='development' webpack",
+    "build": "cross-env NODE_ENV='production' webpack -p",
+    "lint": "standard"
   },
   "devDependencies": {
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
-    "babel-loader": "^7.1.1",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-env": "^1.6.0",
-    "css-loader": "^0.28.4",
-    "eslint": "^4.4.1",
-    "eslint-loader": "^1.9.0",
-    "extract-text-webpack-plugin": "^3.0.0",
-    "file-loader": "^0.11.2",
-    "handlebars": "^1.3.0",
-    "handlebars-loader": "^1.5.0",
-    "html-webpack-plugin": "^2.30.1",
-    "jasmine-core": "^2.7.0",
-    "json-loader": "^0.5.7",
-    "karma": "^1.7.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-jasmine": "^1.1.0",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-webpack": "^2.0.4",
-    "loader-utils": "^1.1.0",
-    "node-sass": "^4.5.3",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.18.2",
-    "url-loader": "^0.5.9",
-    "webpack": "^3.5.1"
+    "jest": "^23.4.0",
```

Comment:
I see, this change was from an earlier recommendation from @hoangtrvu. I think it's a little special in our case, as a scaffold, because all packages we have included are for development, except babel-polyfill

https://docs.npmjs.com/files/package.json#dependencies
>Please do not put test harnesses or transpilers in your dependencies object.